dlpy/network.py

```diff
@@ -810,10 +829,13 @@ def load_weights_from_keras(self, path, labels=False, data_spec=None, label_file
         label_file_name: string, optional
             Fully qualified path to CSV file containing user-defined
             classification labels.  If not specified, ImageNet labels assumed.
+        label_length: int, optional
```

Comment:
Make sure you leave a space before the ':' in docstring lines.  This will mess up Sphinx when parsing.  Make sure to correct others above as well.

Reply:
I will fix and re-submit.